test/unit/util/fs.js

```diff
@@ -94,4 +95,47 @@ describe('FS functions', () => {
       expect(fs.existsSync(newPath)).to.equal(true);
     });
   });
+
+  describe('getFilepathsInFolder', () => {
+    it('should return sorted file list under the directory.', async () => {
+      const directory = await createTemp();
+      fs.writeFileSync(path.join(directory, 'testfile1.txt'), 'testfile1');
```

Comment:
Can you either use `fileHelp` for automatic cleanup (see test/cli/migrate.spec.js for a usage example) or add manual cleanup steps?

Reply:
Wait, I see you are using `createTemp()`. When is it deleted?

Reply:
Checked the implementation, all good here, sorry.